core/Bank.cpp

```diff
@@ -50,8 +50,9 @@ Bank::Bankinit()
 
     // Calculate bank height required for CSL and MDQ...
     // 5u space btw subarrays
-    Bankheight = numberofMemoryArrays * MemoryArrayheight + 
-    (numberofMemoryArrays - 1)*5;
+     Bankheight = numberofMemoryArrays * MemoryArraywidth +
```

Comment:
The variable names don't make sense to me. A _height_ should be calculated with _heights_ not with a _width_.
Maybe for a better understanding there should be a ASCI-ART image that explains what is going on in detail.